transactions/DisruptArt/transferToken.json

```diff
@@ -0,0 +1,9 @@
+{
+    "arguments": [
+        "recpAccount",
+        "tokenIds"
+    ],
+    "messages": {
+        "en": "Token Transfer"
```

Comment:
It'd be better to have more specific message.

Reply:
@scottphc, i have updated the message.